projects/landing-page/src/App.tsx

```diff
@@ -57,21 +57,21 @@ const App: React.FunctionComponent = () => {
               </CardNetwork>
               <CardNetwork
                 title='Rococo'
-                statusProps={{status:'very soon'}}
+                statusProps={{status:'soon'}}
```

Comment:
I would put the `Rococo` (as its the 1 non supported) last

Reply:
Fixed